marlowe-playground-client/src/BlocklyComponent/Types.purs

```diff
@@ -1,19 +1,22 @@
 module BlocklyComponent.Types where
 
 import Prelude hiding (div)
-import Blockly.Internal (BlockDefinition, XML)
 import Blockly.Generator (Generator)
+import Blockly.Internal (BlockDefinition, XML)
 import Blockly.Types as BT
 import Data.Lens (Lens')
 import Data.Lens.Record (prop)
+import Data.List (List)
 import Data.Maybe (Maybe(..))
 import Data.Symbol (SProxy(..))
+import Halogen (SubscriptionId)
 
 type State
   = { blocklyState :: Maybe BT.BlocklyState
     , generator :: Maybe Generator
     , errorMessage :: Maybe String
-    , useEvents :: Boolean
+    , blocklyEventSubscription :: Maybe SubscriptionId
+    , eventsWhileDragging :: Maybe (List BT.BlocklyEvent)
```

Comment:
why not an empty list for Nothing?

Reply:
I thought of that, also creating a `Maybe (NonEmptyList)`. 

I prefer Just/Nothing to distinguish the fact that we are in drag-n-drop rather than relying on the amount of elements that I have. But I'm not married with the idea.

Reply:
ok, it's fine, I suppose they have different meanings here semantically